stores/yaml/store.go

```diff
@@ -309,6 +316,12 @@ func (store *Store) LoadPlainFile(in []byte) (sops.TreeBranches, error) {
 		if err != nil {
 			return nil, fmt.Errorf("Error unmarshaling input YAML: %s", err)
 		}
+		// Prevent use of reserved keywords
+		for _, item := range branch {
+			if item.Key == "sops" {
```

Comment:
Nit: given the string is also matched against on L281. It may be wise to move it to a `const`.

Reply:
There's already a constant for it: `SopsMetadataKey` (in stores/stores.go). That one should be used.